### ISSUE TITLE
Better timestamp picker

### ISSUE THREAD

**Issue by mattbdean** [mattbdean/Helium#9]
Currently, `@angular/material` only supports the date picker. The timestamp format isn't explicit for non-WebKit browsers like Firefox.

One solution is to simply wait until Angular Material implements the datetime picker since it is a [planned feature](https://material.angular.io/components/datepicker/overview#current-state).

See also [Promact/md2](https://github.com/Promact/md2)

**Comment by mattbdean** [comment]
Fixed by [`450f473`](https://github.com/mattbdean/Helium/commit/450f473c8923f76d2e82ee712ae57d40f2e779a7), will close once #11 is merged into master.
